hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/procedure/TestCallCommandParser.scala

```diff
@@ -85,6 +85,17 @@ class TestCallCommandParser extends HoodieSparkSqlTestBase {
     checkParseExceptionContain("CALL cat.system radish kebab")("mismatched input 'CALL' expecting")
   }
 
+  test("Test Call Produce with semicolon") {
```

Comment:
Can we also add a test case for the old mode (missing `;`) to verify the original still works OK.